cmd/hubd/docker-compose.yml

```diff
@@ -14,6 +14,10 @@ services:
       - HUB_ADDR_MONGO_URI
       - HUB_ADDR_IPFS_API=/dns4/ipfs/tcp/5001
       - HUB_ADDR_POWERGATE_API
+      - HUB_BUCKET_MAX_SIZE=1073741824
+      - HUB_BUCKET_MAX_NUMBER_PER_THREAD=10000
+      - HUB_BUCKET_TOTAL_MAX_SIZE=1073741824
+      - HUB_THREAD_MAX_NUMBER_PER_OWNER=100
```

Comment:
Tunable. Discussed a bit with @sanderpick .
Since `HUB_BUCKET_TOTAL_MAX_SIZE` (account-wide limit) came after `HUB_BUCKET_MAX_SIZE` (individual bucket limit) maybe only the first limit makes more sense. Leave the other since doesn't hurt. (Could be 0 to be ignored).